### PR TITLE
Feature/consider feedback

### PR DESCRIPTION
**Pull request by 01Student10**
implemented needed feedback.
added more tests

### CODE REVIEW

File: lib/database/db_creator.dart [hunk 1]
```diff
@@ -43,7 +43,7 @@ class DBCreator {
 
   Future<Database> initDB() async {
     Directory documentsDirectory = await getApplicationDocumentsDirectory();
-    String path = documentsDirectory.path + '/subject_35.db';
+    String path = documentsDirectory.path + '/subject_36.db';
```
Comment (Plateria) [inline lib/database/db_creator.dart:46]: why do we need a new database ¯\_(ツ)_/¯

File: lib/presenter/subject_add.dart [hunk 2]
```diff
@@ -77,22 +78,31 @@ class SubjectAddState extends State<SubjectAdd> {
       setState(() {
         isValidated = true;
       });
+      SnackBar snackBar = SnackBar(
```
Comment (Plateria) [inline lib/presenter/subject_add.dart:81]: I like the colors

File: lib/presenter/subject_add.dart [hunk 3]
```diff
@@ -77,22 +78,31 @@ class SubjectAddState extends State<SubjectAdd> {
...
-                  child: const Text('Okay'),
-                  onPressed: () {
-                    Navigator.of(context).pop(true);
-                  },
-                )
-              ]);
-        },
+      SnackBar snackBar = SnackBar(
+        content: Text(
+          "please check your input again.",
```
Comment (Plateria) [inline lib/presenter/subject_add.dart]: Snackbars should start with a capital letter.

File: lib/view/main_screen.dart [hunk 4]
```diff
@@ -84,6 +84,14 @@ class _MainScreenState extends State<MainScreen> {
     );
   }
 
+  SnackBar snackBar = SnackBar(
```
Comment (Plateria) [inline lib/view/main_screen.dart]: Unnused?

**Review by Plateria — CHANGES_REQUESTED** [review]

File: lib/presenter/subject_add.dart (hunk 3 above)
Comment (01Student10) [reply lib/presenter/subject_add.dart]: fixed it.

File: lib/presenter/subject_add.dart (hunk 2 above)
Comment (01Student10) [reply lib/presenter/subject_add.dart:81]: good

**Comment by 01Student10** [comment]
fixed it.

**Review by Plateria — APPROVED** [review]
I like the new snackbars, that the user gets some feedback.

**Pull request merged**